lib/vmfloaty/ssh.rb

```diff
@@ -30,7 +30,12 @@ def self.ssh(verbose, host_os, token, url)
       end
 
       hostname = "#{response[host_os]["hostname"]}.#{response["domain"]}"
-      cmd = "#{ssh_path} #{user}@#{hostname}"
+      ssh_options= ["-oStrictHostKeyChecking=no", "-oUserKnownHostsFile=/dev/null"]
+      if ENV['VMFLOATY_SSH_KEY']
+        ssh_options << "-i #{ENV['VMFLOATY_SSH_KEY']}"
```

Comment:
Hey just checking in on this @underscorgan . Lemme know what you think about my suggestion if you have a free moment, thank you!